web/src/screens/Dashboard/Dashboard.tsx

```diff
@@ -5,7 +5,7 @@ import FormInput from '../../components/LoginModal/FormInput';
 import ScoreCard from '../../components/LoginModal/ScoreCard';
 import { mockCandidates, mockCriteria } from '../../data';
 
-const Home = () => {
+const Dashboard = () => {
   const [showModal, setShowModal] = useState<boolean>(false);
 
   const showModalHandler = () => {
```

Comment:
Let's keep it consistent with the naming of actions. How about 'handleModalDisplay'?